var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -354,6 +348,8 @@ def hostconfig(self, spec, prefix):
         if cxxflags:
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS", cxxflags))
         fflags = ' '.join(spec.compiler_flags['fflags'])
+        if '%cce' in self.spec:
+            fflags += "-ef"
```

Comment:
This needs a space in the front.

Reply:
This will start with `-some-flag -another-flag` from line 350 then end up with `-some-flag -another-flag-ef` (notice the lack of a space)